mlflow/store/db_migrations/versions/39d1c3be5f05_test.py

```diff
@@ -0,0 +1,27 @@
+"""test
```

Comment:
Is this the right approach to apply [this change](https://github.com/mlflow/mlflow/pull/4180/files#diff-a7bffc8e5a9a4dd89c47d6f5bd00fd9dbb9f0bd25a82921d5a95f122b4ad3e39R368)?

Reply:
I'll rename this file to a more desriptive name

Reply:
Looks good! Can we rename the revision (regenerating the revision hash if necessary) to `add_is_nan_constraint_for_latest_metrics_if_necessary` or something similar (feel free to shorten it if that's verbose). 

Can we also insert a comment in the migration explaining that this is only relevant for what I would presume is a small set of users who installed sqlalchemy 1.4.0 with MLflow <= 1.14.1? For all other users (folks with preexisting databases or older sqlalchemy versions), this is a no-op, right?